packages/lodestar/src/sync/initial/fast/index.ts

```diff
@@ -113,7 +113,7 @@ export class FastSync
 
   private setBlockImportTarget = (fromSlot?: Slot): void => {
     const lastTarget = fromSlot || this.blockImportTarget;
-    const newTarget = this.getNewBlockImportTarget(this.blockImportTarget);
+    const newTarget = this.getNewBlockImportTarget(lastTarget);
```

Comment:
avoid calling too big range like `{"startSlot":"509","count":"1472","step":"1"}` in my sync